doc/guide/packages.xml

```diff
@@ -253,4 +253,50 @@ mypackage/test.min.js.gz
 
   </section>
 
+  <section id="package-replace">
+    <title>Replacing an existing package</title>
+
+    <para>If the functionality in a package replaces one of the default packages
```

Comment:
This relates to all packages, right? Not just `default` packages, whatever those are. I suggest ... `replaces that of another package`